db/migrate/20241002215949_add_asset_processing_count_to_episode_delivery_status.rb

```diff
@@ -0,0 +1,5 @@
+class AddAssetProcessingCountToEpisodeDeliveryStatus < ActiveRecord::Migration[7.2]
+  def change
+    add_column :apple_episode_delivery_statuses, :asset_processing_attempts, :integer, default: 0, null: false
```

Comment:
Huh, for some reason I thought we were using the Pipeline states to track how many times we failed to process.  Is this just another way to track it?

Reply:
Sorry, this looks like a rebasing artifact. This landed in the PR https://github.com/PRX/feeder.prx.org/pull/1116 that you reviewed earlier.

Reply:
https://github.com/PRX/feeder.prx.org/pull/1116 is the start of an approach to count the timeouts, per discussion in slack etc counting the error states swallows other non-timeout errors.  Is this what you wanted to do instead?

Reply:
Ah, I thought past-Ryan had seen a migration for something like this before!

This approach works - just wanted to make sure we weren't counting it twice.